media-libs/avisynthplus/avisynthplus-3.7.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="AviSynth+ scripting frameserver"
+DESCRIPTION="A powerful nonlinear scripting language for video."
+HOMEPAGE="https://avs-plus.net"
+COPYRIGHT="2000-2015 Ben Rudiak-Gould, et al., \
+2013-2021 AviSynth+ Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/AviSynth/AviSynthPlus/archive/v$portVersion.tar.gz"
+SOURCE_FILENAME="avisynthplus-$portVersion.tar.gz"
+SOURCE_DIR="AviSynthPlus-$portVersion"
+CHECKSUM_SHA256="8906d9e46dc90a194413d69b710e3106c0aafddc0c5c62004885d0c3beb79862"
+#PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	avisynthplus$secondaryArchSuffix = $portVersion
+	lib:libavisynth = $portVersion
+	lib:libconvertstacked = $portVersion
+	lib:libtimestretch = $portVersion
+	lib:libimageseq = $portVersion
+	lib:libshibatch = $portVersion
+	"
+REQUIRES="
+    haiku$secondaryArchSuffix
+    devil$secondaryArchSuffix
```

Comment:
haikuporter in strict mode will try to tell which libs are really required. devil$secondaryArchSuffix isn't really required, but for instance lib:libIL$secondaryArchSuffix might be